app/models/delivery.js

```diff
@@ -20,6 +20,21 @@ export default DS.Model.extend({
     let adapter = this.store.adapterFor(this.constructor.modelName);
     return adapter.send(this.get('id'), force).then(this.updateAfterAction.bind(this));
   },
+  preview(props) {
+    let adapter = this.store.adapterFor(this.constructor.modelName);
+    let details = {
+      from_user_id: this.get('fromUser.id'),
+      to_user_id: this.get('toUser.id'),
+      project_id: this.get('transfer.project.id'),
+      transfer_id: this.get('transfer.id'),
+      user_message: this.get('userMessage')
+    };
+    for(let prop in props) {
+      details[prop] = props[prop];
+    }
```

Comment:
Eventually you will be able to collapse this loop into `Object.assign(details, props)`.
For now there is a ember polyfill
https://emberjs.com/api/ember/2.18/functions/@ember%2Fpolyfills/assign